notebooks/nessie-delta-demo-nba.ipynb

```diff
@@ -0,0 +1,642 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Nessie Spark SQL Demo with NBA Dataset\n",
+    "============================\n",
+    "This demo showcases how to use Nessie Python API along with Spark3 from Iceberg\n",
+    "\n",
+    "Initialize Pyspark\n",
+    "----------------------------------------------\n",
+    "To get started, we will first have to do a few setup steps that give us everything we need\n",
+    "to get started with Nessie. In case you're interested in the detailed setup steps for Spark, you can check out the [docs](https://projectnessie.org/tools/spark/)\n",
+    "\n",
+    "The Binder server has downloaded spark and some data for us as well as started a Nessie server in the background. All we have to do is start Spark\n",
+    "\n",
+    "The below cell starts a local Spark session with parameters needed to configure Nessie. Each config option is followed by a comment explaining its purpose.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import findspark\n",
+    "from pyspark.sql import *\n",
+    "from pyspark import SparkConf\n",
+    "findspark.init()\n",
+    "\n",
+    "warehouse = 'file://' + os.getcwd() + '/spark_warehouse/delta'\n",
+    "conf = SparkConf()\n",
+    "conf.set(\"spark.jars.repositories\", \"https://storage.googleapis.com/nessie-maven\") # we add our custom fork of delta to the known repositories\n",
```

Comment:
Feels that putting the comments above the `conf.set` makes it more readable in the notebook

Reply:
done